### PR TITLE
[Document-7870][docs] Modify the document system screenshot

### PR DESCRIPTION
**Pull request by amaoisnb**
- Upgrade 3.0 and dev branch screenshot to laster
- Upgrade Workflow Definition page

### CODE REVIEW

**Comment by zhongjiajie** [comment]
thanks, and will task a look tonight

**Comment by zhongjiajie** [comment]
Hi @amaoisnb we only need to use the screenshot in English in our doc, so you should change the file https://raw.githubusercontent.com/apache/dolphinscheduler/12fdd9bbd9d15dfe2a993731a2b5054ea5eb3956/docs/docs/zh/guide/start/quick-start.md and make them using the same picture in English version document.

and remove the exists img in path `img/dag1.png` and `img/task-log.png`

**Comment by zhongjiajie** [comment]
@amaoisnb change LGTM, but could you resolve the conflicts

**Comment by amaoisnb** [comment]
> @amaoisnb change LGTM, but could you resolve the conflicts
Okay, I'll try

**Comment by zhongjiajie** [comment]
@amaoisnb I solve conflict by myself, hope you do not mind

File: docs/docs/zh/guide/start/quick-start.md [hunk 1]
```diff
@@ -50,11 +50,11 @@
   * 点击工作流定义->创建工作流定义->上线工作流定义
 
 <p align="center">
-   <img src="../../../../img/dag1.png" width="60%" />
+   <img src="../../../../img/process_definition_en.png" width="60%" />
```
Comment (EricGao888) [inline docs/docs/zh/guide/start/quick-start.md:53]: As we have discussed here before: https://github.com/apache/dolphinscheduler/issues/10257#issuecomment-1175126227, what about renaming `process_definition_en` to `workflow_definition_en`?

Comment (zhongjiajie) [reply docs/docs/zh/guide/start/quick-start.md:53]: do you mean rename `process_definition_en` to `process_definition` ?

Comment (zhongjiajie) [reply docs/docs/zh/guide/start/quick-start.md:53]: > do you mean rename `process_definition_en` to `process_definition` ?

oh, my bad, you mean `process_definition` vs `workflow` ? we should do it, but I think it is better to do it in separate PR, and change other related content, WDYT

Comment (EricGao888) [reply docs/docs/zh/guide/start/quick-start.md:53]: Sure, sounds good to me.

**Pull request closed without merging**